### DISCUSSION TITLE
Welcome to Foodium Discussions!

### DISCUSSION THREAD

**Discussion by PatilShreyas** [PatilShreyas/Foodium#53] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anandchoudhary-rccl** [comment]


**Reply by PatilShreyas** [reply]
Not in this repo. There's another repo which has compose multiplatform usage for Android and iOS both platforms: https://github.com/PatilShreyas/Foodium-KMM
